.github/workflows/ci-e2e-cassandra.yml

```
@@ -48,6 +48,13 @@ jobs:
       run: bash scripts/cassandra-integration-test.sh ${{ matrix.version.major }} ${{ matrix.version.schema }} ${{ matrix.jaeger-version }}
       env:
         SKIP_APPLY_SCHEMA: ${{ matrix.create-schema == 'auto' && true || false }}
+     
+    - name: Upload Metrics Artifact
+      uses: actions/upload-artifact@v3
+      with:
+              name: cassandra-metrics-${{ matrix.version }}
+              path: ./metrics/cassandra_metrics.txt
```

this job runs in a matrix multiple times. In v1 runs there won't be any metrics. In v2 I don't think we want to upload them with the same file name.
```suggestion
              path: ./metrics/cassandra_metrics_${{ matrix.version.major }}_${{ matrix.version.schema }}_${{ matrix.jaeger-version }}.txt
```

---

@yurishkuro the artifacts will identified with name parameter so we can have simple path for the file ?

---

please implement downloading them, then we can see